waku/v2/node/jsonrpc/admin_api.nim

```diff
@@ -64,34 +64,38 @@ proc installAdminApiHandlers*(node: WakuNode, rpcsrv: RpcServer) =
 
     if not node.wakuRelay.isNil:
       # Map managed peers to WakuPeers and add to return list
-      wPeers.insert(node.peerManager.peers(WakuRelayCodec)
+      wPeers.insert(node.peerManager.peerStore
+                                    .peers(WakuRelayCodec)
                                     .mapIt(WakuPeer(multiaddr: constructMultiaddrStr(toSeq(it.addrs.items)[0], it.peerId),
                                                     protocol: WakuRelayCodec,
-                                                    connected: node.peerManager.connectedness(it.peerId) == Connectedness.Connected)),
+                                                    connected: it.connection == Connectedness.Connected)),
```

Comment:
This should be `it.connectedness`, given that you are comparing it with an enum named `Connectedness`.